README.md

```diff
@@ -41,6 +41,43 @@ bin/console analyze:php:code-compliance-report
 ​
 This command returns the content of a previously generated report. Alternatively, you can view a report via an editor.
 ​
+
+
+### Define custom prefixes for core entity names
+
+When evaluator checks project-level code entities for existing and potential matches with the core ones, it skips the entities that have the `Pyz` prefix in their name. Such are considered unique and will not conflict with core entities in future because there will never be an entity with the `Pyz` prefix in the core.
+
+When solving upgradability issues, you can use the `Pyz` prefix to make your entities unique. To use custom prefixes, do the following:
+
+1. Create `/tooling.yaml`
+2. Define `Pyz` and needed custom prefixes. For example, define `Zyp` as a custom prefix:
+
+```yaml
+evaluator:
```

Comment:
As I remember it had a key `upgrader` before, right? So we can't just change the key name here, it will produce a breaking change for the projects that use this configuration already.

Reply:
OK, for protocol - we've discussed that the previous key `upgrader` was selected wrong. "Prefix" configuration relates also to the `evaluator` tool. So we accept breaking change here because the potential expected impact at the moment is 1 or 2 projects.

Reply:
++